src/views/Home.vue

```diff
@@ -47,7 +59,29 @@ export default {
 		BaseInfoWidget,
 		// PriceChartWidget,
 		RecentBlocksWidget,
-		RecentTransactionsWidget
+		RecentTransactionsWidget,
+		RecentListWidget
+	},
+
+	data() {
+		return {
+			recentListWidgetSetup: [
+				{
+					title: 'namespaces',
+					listingPage: '/namespaces',
```

Comment:
Changing this field to `viewMoreLinkURL` could add more sense, what do you think?